srcpkgs/containerd/template

```diff
@@ -20,7 +20,7 @@ maintainer="Paul Knopf <pauldotknopf@gmail.com>"
 license="Apache-2.0"
 homepage="https://github.com/containerd/containerd"
 distfiles="https://github.com/containerd/containerd/archive/v${version}.tar.gz"
-checksum=e381c5133feacf7a9d6991c3535103f3c1f7a86b5b8ce2df226c5abde77fb5d8
+checksum=@7121be3c9cf08cb1d922e9dc858f7ea00439fb0eb732776412e598fa40a77451
```

Comment:
I don't understand why switch to content checksums.

Reply:
Out of habit I use all options to `xgensum` when I update checksums.  I guess I didn't understand the `-c` option.  I switched it to SHA256 checksum.